scripts/yse/yse.py

```diff
@@ -0,0 +1,264 @@
+                  Rojas-Bravo, Cesar and
+                  Siebert, Matthew R. and
+                  Smith, Ken W and
+                  de Soto, Kaylee M. and
+                  Stroh, Michael C. and
+                  Tinyanont, Samaporn and
+                  Taggart, Kirsty and
+                  Ward, Sam M. and
+                  Wojtak, Radosław and
+                  Auchettl, Katie and
+                  Blanchard, Peter K. and
+                  de Boer, Thomas J. L. and
+                  Boyd, Benjamin M. and
+                  Carroll, Christopher M. and
+                  Chambers, Kenneth C. and
+                  DeMarchi, Lindsay and
+                  Dimitriadis, Georgios and
+                  Dodd, Sierra A. and
+                  Earl, Nicholas and
+                  Farias, Diego and
+                  Gao, Hua and
+                  Gomez, Sebastian and
+                  Grayling, Matthew and
+                  Grillo, Claudia and
+                  Hayes, Erin E. and
+                  Hung, Tiara and
+                  Izzo, Luca and
+                  Khetan, Nandita and
+                  Kolborg, Anne Noer and
+                  Law-Smith, Jamie A. P. and
+                  LeBaron, Natalie and
+                  Lin, Chien C. and
+                  Luo, Yufeng and
+                  Magnier, Eugene A. and
+                  Matthews, David and
+                  Mockler, Brenna and
+                  O'Grady, Anna J. G. and
+                  Pan, Yen-Chen and
+                  Politsch, Collin A. and
+                  Raimundo, Sandra I. and
+                  Rest, Armin and
+                  Ridden-Harper, Ryan and
+                  Sarangi, Arkaprabha and
+                  Schrøder, Sophie L. and
+                  Smartt, Stephen J. and
+                  Terreran, Giacomo and
+                  Thorp, Stephen and
+                  Vazquez, Jason and
+                  Wainscoat, Richard and
+                  Wang, Qinan and
+                  Wasserman, Amanda R. and
+                  Yadavalli, S. Karthik and
+                  Yarza, Ricardo and
+                  Zenati, Yossef},
+  title        = {{The Young Supernova Experiment Data Release 1 (YSE 
+                   DR1) Light Curves}},
+  month        = nov,
+  year         = 2022,
+  publisher    = {Zenodo},
+  version      = {1.0.0},
+  doi          = {10.5281/zenodo.7317476},
+  url          = {https://doi.org/10.5281/zenodo.7317476}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+Time-series dataset from the Young Supernova Experiment Data Release 1 (YSE DR1).
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://zenodo.org/records/7317476"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY 4.0"
+
+_VERSION = "0.0.1"
+
+_STR_FEATURES = [
+    "object_id",
+    "spec_class",
```

Comment:
is this object type? do we want to unify this colname (i'm currently calling it `obj_type`, but open to changing)? also, is this data a string like "SNIa"? wondering since typically SNANA outputs some integer eg 101 for SNIa

Reply:
Ah for YSE, the spec_class (which can defo be renamed to obj_type, although it may be nice to have spec_class and phot_class separately in the future... ?) is a string in the YSE data (and in similarly formatted data, like Foundation)

Reply:
Agreed to change classification feature key to `obj_type`. @benboyd97 @ado8

Reply:
this doesn't seem to have changed